app/pb/account.proto

```diff
@@ -77,3 +98,49 @@ message GetContributorFormInfoRes {
 message MarkContributorFormFilledReq {
   bool filled_contributor_form = 1;
 }
+
+message ChangePhoneReq {
+  // New phone number in E.164 format, or empty string to remove phone number.
+  // The phone number will be visible to moderators in your area, verified or not.
+  string phone = 1;
+
+  // If you have a previously verified number, changing your number and losing
+  // the verified status will fail if false is passed.
+  bool remove_verification = 2;
+}
+
+message SendVerificationSMSReq {
+}
+
+message SendVerificationSMSRes {
+  enum SendVerificationSMSStatus {
+    SMS_RESULT_CODE_SENT = 0;
+    SMS_RESULT_ALREADY_VERIFIED = 1;  // No code sent since the same number is already verified, and a renewal can't be done yet
+    SMS_RESULT_RATELIMIT = 2;   // Hold your horses, contact admin to send more than one SMS in a short period
+    SMS_RESULT_UNSUPPORTED_OPERATOR = 3;  // Sorry, we don't know how to send SMS to your number
+    SMS_RESULT_OPERATOR_ERROR = 4;  // Unknown, see operator_error_message.
+    SMS_RESULT_CONSENT_NEEDED = 5;  // Accept a later ToS version to allow couchers to use external services for sms delivery
+    SMS_RESULT_NO_PHONE_NUMBER = 6;  // User hasn't inputed a phone number yet
```

Comment:
Error codes should be returned through gRPC StatusCodes. Also, `SMS_RESULT_CONSENT_NEEDED` is not needed, this API is "secure" so not having agreed to latest TOS means the user can never get this far.

Reply:
Ok, I will spread them out evently over INVALID_ARGUMENT, FAILED_PRECONDITION, RESOURCE_EXHAUSTED and ALREADY_EXISTS.